cypress/utils/apps/query_enhancements/commands.js

```diff
@@ -101,6 +101,7 @@ Cypress.Commands.add('setQueryEditor', (value, options = {}) => {
       .should('be.visible')
       .wait(200)
       .type(escape ? `${value}{esc}` : value, {
+        delay: 40,
```

Comment:
i noticed that the tests were sometimes failing locally because it wouldn't fully type out the query string it is supposed to type. Sometimes it would skip a letter or so on. It might be related to the monaco update? The default delay is 10, i upped it to 40ms.

Reply:
I remember this was a common issue with Cypress typing. Usually, playing around with the delay settings work, but I used to rerun the tests a few hundred times to ensure that it won't be flaky. I'll monitor the Cypress tests on our CI; if they continue to be flaky on typing, I think we should revisit this to figure out a consistent solution.